source/Applications/Basic/FileFormats/ReadIterateZDF.m

```diff
@@ -1,7 +1,12 @@
 % Import ZDF point cloud.
 
+% Adding directory that contains zividApplication to search path.
+addpath(genpath([fileparts(fileparts(fileparts(pwd))),filesep,'Camera',filesep,'Basic']));
```

Comment:
This is not right I believe, pwd is current folder, but we don't have to be in this folder to run the script. Then it will fail, the correct way to get path of file is:
mfilename('fullpath')

Reply:
deon